plugins/homekit/src/types/garage.ts

```diff
@@ -22,9 +22,6 @@ addSupportedType({
 
         let targetState = !!device.entryOpen ? Characteristic.TargetDoorState.OPEN : Characteristic.TargetDoorState.CLOSED;
         service.getCharacteristic(Characteristic.TargetDoorState)
-            .on(CharacteristicEventTypes.GET, (callback: NodeCallback<CharacteristicValue>) => {
```

Comment:
is this deletion intentional? not sure why garage is being changed.

Reply:
I made a comment in the pr description. Those are unneeded, as we are calling the callback twice per event. Once from bind characteristic and then again here. But the bind characteristic handling is much better. The double callback is causing warning all over the logs.